test/Microsoft.AspNetCore.Mvc.Core.Test/DependencyInjection/ApplicationModelConventionExtensionsTest.cs

```diff
@@ -13,6 +13,34 @@ namespace Microsoft.Extensions.DependencyInjection
     public class ApplicationModelConventionExtensionsTest
     {
         [Fact]
+        public void DefaultParameterModelConvention_AppliesToAllParametersInApp()
+        {
+            // Arrange
+            var app = new ApplicationModel();
+            app.Controllers.Add(new ControllerModel(typeof(HelloController).GetTypeInfo(), new List<object>()));
+            app.Controllers.Add(new ControllerModel(typeof(WorldController).GetTypeInfo(), new List<object>()));
+
+            var options = new MvcOptions();
+            options.Conventions.Add(new SimpleParameterConvention());
+
+            // Act
+            options.Conventions[0].Apply(app);
+
+            // Assert
+            foreach (var controller in app.Controllers)
+            {
+                foreach (var action in controller.Actions)
+                {
+                    foreach (var parameter in action.Parameters)
+                    {
+                        Assert.Contains("TestProperty", parameter.Properties.Keys);
+                        Assert.Contains("TestValue", parameter.Properties.Values);             
```

Comment:
btw **the** way to do this kind of thing is with `Assert.Collection` - learn it and 😍 it. This isn't really wrong, but `Assert.Collection` does exactly what you want. Or in this case `Assert.Single` would be great too.

```
Assert.Collection(parameter.Properties, kvp => kvp.Key == "..." && kvp.Value == "...");
```
**or**
```
var kvp = Assert.Single(parameter.Properties);
Assert.Equal("...", kvp.Key);
Assert.Equal("...", kvp.Value);
```